src/fragments/start/getting-started/flutter/generate-model.mdx

```diff
@@ -1,68 +1,96 @@
-With the basic setup complete, you can now model the data your application will work with. Amplify DataStore persists the modeled data on your local device and can even synchronize to a backend API without writing any additional code. These models are specified as [GraphQL](http://graphql.org/) schemas. You can [learn more](/cli/graphql/overview) about GraphQL schemas and data modeling. For now, let’s start by generating some initial data models.
+With the basic setup complete, you can now model the data your application will work with. 
+These models are specified as  [GraphQL](http://graphql.org/) schemas. You can 
+[learn more](/cli/graphql/overview) about GraphQL schemas and data modeling. For now, 
+let's start by creating our budget management data model.
```

Comment:
in compliance with the style guide, can we remove "our" from here.

Let us use "We can start by creating the budget management data model".